src/app/(private)/reservas/[id]/components/detail-booking-card.tsx

```
@@ -105,64 +120,44 @@ export function DetailBookingCard({ bookingId }: { bookingId: string }) {
-                                                            }
-                                                            key={adjustment.id}
-                                                            label={
-                                                                adjustment.label
-                                                            }
-                                                            value={
+                                        <div>
+                                            {product.adjustments.map(
+                                                (adjustment) => (
+                                                    <>
+                                                        <p>
+                                                            {adjustment.label} -{" "}
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)